src/pyscript/_generator.py

```diff
@@ -111,12 +111,15 @@ def file_to_html(
 
 
```

Comment:
Maybe the method below could be factored into 2 internal methods? One for the regular create and one for the wrapping case? Then there would be a single switch for 'if wrap'...